.github/workflows/cross-browser-test.yml

```diff
@@ -0,0 +1,34 @@
+name: Cross-Browser Test Flow
+
+on:
+  push:
```

Comment:
You probably want `[push, pull_request]` here as well.

Reply:
Fixed.

Reply:
Oh wait, I think we only had it run on push because we have Saucelabs concurrency limitations right now that we are working to get fixed.  I'm switching it back to push.